shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/TripInstantDisplay.kt

```diff
@@ -115,6 +121,19 @@ sealed class TripInstantDisplay {
                 }
             }
             val predictionTime = prediction.stopTimeAfter(now)
+            if (allowTimeWithStatus && prediction.status != null) {
```

Comment:
question: I see `allowTimeWithStatus` only is for the stop details view - is there any reason we wouldn't want to show a delayed scheduled CR trip in nearby transit too?